tests/core/test_actions.py

```diff
@@ -510,6 +510,21 @@ async def test_action_session_start_without_slots(
     assert events == [SessionStarted(), ActionExecuted(ACTION_LISTEN_NAME)]
 
 
+async def test_action_session_start_with_metadata(
```

Comment:
I don't think this test is necessary now.

Reply:
Done!